mirascope/integrations/logfire/_utils.py

```diff
@@ -186,3 +186,55 @@ async def handle_structured_stream_async(
     span_data = get_structured_stream_span_data(result)
     span_data["async"] = True
     logfire_span.set_attributes(span_data)
+
+
+async def handle_error(
```

Comment:
we should refactor these to use a shared `_get_error_span_data` utility method

also, if you'd be so kind as to update all of the similar utilities to be private as well (e.g. `_get_call_response_span_data`) that would be awesome :)

Reply:
Yes it's done 💯